Update Simplified Chinese translation

itflake opened the pull request:
Update bundle_zh_CN.properties

liangliangsan commented:
is a bad PR, please don't merge it

itflake commented:
> is a bad PR, please don't merge it

reason?

BugTD commented:
> > is a bad PR, please don't merge it
> 
> reason?

very bad,you just open many PRs to hide the opposite views

itflake commented:
To summarize:
I didn't make any translation mistakes, I just changed the old translations which made old players unhappy.
I don't care how many negative reviews there are. I have fixed them all.
At the same time, new PRs are opened for real problems to comment not to let useless comments bury useless comments.

itflake commented:
> > > is a bad PR, please don't merge it是一个糟糕的 PR，请不要合并它
> > 
> > 
> > reason?  原因？
> 
> Too many unnecessary changes

Not needed not equal cannot be improved.

liangliangsan commented:
> > is a bad PR, please don't merge it
> 
> reason?

you had 3 PRs and closed them yourself, but the previous criticism has not disappeared

itflake commented:
> > > is a bad PR, please don't merge it
> > 
> > 
> > reason?
> 
> you had 3 PRs and closed them yourself, but the previous criticism has not disappeared

As mentioned above, I fixed it.
As for the previous ones, I don't care at all.

BugTD commented:
> > > > is a bad PR, please don't merge it
> > > 
> > > 
> > > reason?
> > 
> > 
> > you had 3 PRs and closed them yourself, but the previous criticism has not disappeared
> 
> As mentioned above, I fixed it.如上所述，我修复了它。 As for the previous ones, I don't care at all.至于前面的，我一点也不在乎。

so you don't care about my review right?

itflake commented:
> > > > > is a bad PR, please don't merge it
> > > > 
> > > > 
> > > > reason?
> > > 
> > > 
> > > you had 3 PRs and closed them yourself, but the previous criticism has not disappeared
> > 
> > 
> > As mentioned above, I fixed it.如上所述，我修复了它。 As for the previous ones, I don't care at all.至于前面的，我一点也不在乎。
> 
> so you don't care about my review right?

I mean that all the changes that should be restored have been restored. I mean that the remaining changes that have no impact should be left alone. Why are you holding me back?

BugTD commented:
所有人都认为翻译质量很差，都讨厌这种翻译。每个人都基于自己长期的游戏游玩和上网经验。我接触到的几乎30位资深玩家全部反对你的翻译，新玩家看到你的翻译也只会忽略他



高处不胜寒，寒中风凛冽

way-zer commented:
为什么你不在旧的PR解决问题，而是反复新开？

itflake commented:
> 为什么你不在旧的PR解决问题，而是反复新开？ Why don't you fix the problem in the old PR instead of keeping opening new ones?

评论太多了，而且反复改了好几次。所以重新开一个最完美的提交。

AWA-SUBARU-Deisu commented:
为什么你要不断做出你认为是正确的改动，哪怕几乎整个中国玩家群体都在反对？

like-cookie commented:
> > > 为什么你不在旧的PR解决问题，而是反复新开？ Why don't you fix the problem in the old PR instead of keeping opening new ones?为什么你不在旧的 PR 解决问题，而是反复新开？为什么不修复旧 PR 中的问题，而不是一直打开新的 PR 呢？
> > 
> > 
> > 评论太多了，而且反复改了好几次。所以重新开一个最完美的提交。
> 
> I don't think so, just want to avoid the problem.我不这么认为，我只是想避免这个问题。

like-cookie commented:
> > 为什么你不在旧的PR解决问题，而是反复新开？ Why don't you fix the problem in the old PR instead of keeping opening new ones?为什么你不在旧的 PR 解决问题，而是反复新开？为什么不修复旧 PR 中的问题，而不是一直打开新的 PR 呢？
> 
> 评论太多了，而且反复改了好几次。所以重新开一个最完美的提交。

I don't think so, just want to avoid the problem.

BugTD commented:
> > 为什么你不在旧的PR解决问题，而是反复新开？ Why don't you fix the problem in the old PR instead of keeping opening new ones?
> 
> 评论太多了，而且反复改了好几次。所以重新开一个最完美的提交。

所以这就是你改空格，标点和语序的原因？

BugTD commented:
You never had Chinese players in your sights from the very beginning. To be precise, you were either overly arrogant or excessively "anti-traditional." The issues you raised didn’t include any logs, making them impossible to reproduce. You submitted four translation PRs in a row but still failed to address some of the criticized problems. You kept saying you were translating wholeheartedly—so why didn’t you discuss with Chinese players in the translation discussion group, instead of spamming garbage PRs on GitHub? You accused us of not understanding and having narrow vision, but you didn’t even bother to consult a few veteran players. Multiple veteran players have criticized or even reprimanded you, yet you refused to listen. You didn’t even read my reviews. You obsess over a few spaces, punctuation marks, and word order. So I can only reasonably suspect that everything you’re doing is just a fantasy about making it onto Mindustry’s contributor list—after all, that’s your ultimate goal, isn’t it?

liangliangsan commented:
> > > > > To summarize: I didn't make any translation mistakes, I just changed the old translations which made old players unhappy. I don't care how many negative reviews there are. I have fixed them all. At the same time, new PRs are opened for real problems to comment not to let useless comments bury useless comments.
> > > > 
> > > > 
> > > > You have not made any improvements, and in previous disputes, most people generally believe that you are not qualified to be a Mindustry translator
> > > 
> > > 
> > > Honestly, I put my heart and soul into every update and fix the issues seriously, so you are just following the trend without seeing the facts clearly.
> > 
> > 
> > Your passion can't change the fact that most people criticize the poor translation, and players shouldn't pay for it.
> 
> No one says the translation quality has deteriorated, they just don't like it because they follow the trend, and so do you. Please provide your evidence.
> 
> I promise that my translation will not affect old players, but will be more friendly and better to new players.

You've opened 3 PRs before, so I don't think I need to repeat other reviewers' comments

itflake commented:
关于你们所说的质量差的问题本质上是乌龙。

> link.github.description = 游戏源代码
> discord = 加入 Discord 聊天室！

像这种改动基本没用明显的优劣之分，我的改动只是让它更精简了。

> unlocked = 已解锁新内容！
> unlocked = 解锁了新内容！

现有翻译就是我改的，我只是还原到以前的，而且新的更合适，更简单易懂。

> wave.guardianwarn = Boss 将在 [accent]{0}[] 波后抵达
> wave.guardianwarn.one = Boss 将在 [accent]{0}[] 波后抵达

这个改动是群友推荐的。

> objective.build = [accent]建造[] [lightgray]{0}[] 个 {1} [lightgray]{2}
> objective.build = [accent]建造：[][lightgray]{0}[]x\n{1}[lightgray]{2}

这个是我上次埋的坑，这次修复了。

> sectors.survives = [accent]撑过 {0} 波攻击
> sectors.survives = [accent]抵御 {0} 波攻击

这个也是群友推荐改掉原来的。

> rules.unitcapvariable = 单位上限随核心数量增加
> rules.unitcapvariable = 核心可增加单位上限

描述成“核心可增加”要比“跟随核心数量”更准确，因为每种核心增加的上限不同。

...

我的改动大多数都是非常好的。一部分意义不大，但这些无需争论。

BugTD commented:
> > > > > > > To summarize: I didn't make any translation mistakes, I just changed the old translations which made old players unhappy. I don't care how many negative reviews there are. I have fixed them all. At the same time, new PRs are opened for real problems to comment not to let useless comments bury useless comments.
> > > > > > 
> > > > > > 
> > > > > > You have not made any improvements, and in previous disputes, most people generally believe that you are not qualified to be a Mindustry translator
> > > > > 
> > > > > 
> > > > > Honestly, I put my heart and soul into every update and fix the issues seriously, so you are just following the trend without seeing the facts clearly.
> > > > 
> > > > 
> > > > Your passion can't change the fact that most people criticize the poor translation, and players shouldn't pay for it.
> > > 
> > > 
> > > No one says the translation quality has deteriorated, they just don't like it because they follow the trend, and so do you. Please provide your evidence.
> > > I promise that my translation will not affect old players, but will be more friendly and better to new players.
> > 
> > 
> > You've opened 3 PRs before, so I don't think I need to repeat other reviewers' comments
> 
> > You never had Chinese players in your sights from the very beginning. To be precise, you were either overly arrogant or excessively "anti-traditional." The issues you raised didn’t include any logs, making them impossible to reproduce. You submitted four translation PRs in a row but still failed to address some of the criticized problems. You kept saying you were translating wholeheartedly—so why didn’t you discuss with Chinese players in the translation discussion group, instead of spamming garbage PRs on GitHub? You accused us of not understanding and having narrow vision, but you didn’t even bother to consult a few veteran players. Multiple veteran players have criticized or even reprimanded you, yet you refused to listen. You didn’t even read my reviews. You obsess over a few spaces, punctuation marks, and word order. So I can only reasonably suspect that everything you’re doing is just a fantasy about making it onto Mindustry’s contributor list—after all, that’s your ultimate goal, isn’t it?
> 
> I don't submit logs because there are no errors and no logs. I have uploaded all the proofs of save and screenshots.我不提交日志，因为没有错误，也没有日志。我已经上传了所有保存和屏幕截图的证明。

so you just answered the smallest quesiton

AWA-SUBARU-Deisu commented:
你拿不出哪怕一张截图去证明有人同意你的翻译改动

Routerteaching-179 commented:
。瞎改很光荣是吧

AWA-SUBARU-Deisu commented:
从你的行为我只看到了一个在意淫贡献者头衔而不去考虑他人感受的自私自利者

Routerteaching-179 commented:
你瞎改很光荣，是吧

itflake commented:
> 你瞎改很光荣，是吧

我已经给了很多需要翻译的理由，这个 PR 即使不通过，未来也会有人根据这个会修的。

BugTD commented:
> > 我相信你穿越到战锤40K世界一定会成为一名绿皮，因为你总是能发挥“我寻思”的力量而拿不出真实存在的证据
> 
> ~我相信你穿越到异世界一定会成为一名炮灰，因为你总是跟随大流而看不出主角有多强。~

所以从大流一定是错的？你怎么看出来从大流的？你到底有没有回答过哪怕一个重要的问题？你以为自己很幽默？你是不是认为自己是一个很厉害的人，别人没注意到的改动你都注意到了？你是不是认为自己是mdt大师所以不用询问任何人的意见？如果你不在中午不联网说话，那么为什你有权利翻译中文？

BugTD commented:
你到底有没有看懂我发给你的英文，还是说你的英文都是机翻所以看不懂？

BugTD commented:
那我给你原文好了：
你从一开始就没有把中国玩家放在眼里。确切的说你过于傲慢或者过于“反传统”，你提的issue没有提供任何日志，无法复现。你的翻译pr连续提了4个但是仍未修改其中一些为人诟病的问题，你一直在说你全心全意的在翻译——那你为什么不和中文玩家在翻译交流群中讨论，反而只在github上不断的发垃圾pr呢？你指责我们不懂，眼光短浅，但是你没有询问哪怕几个资深玩家的意见。多位资深玩家曾经批评甚至指责过你，也不听。你甚至不会看我的review，你执着于几个空格和标点，语序。那么我只能有理由怀疑你做的一切都是在意淫自己能够进mindustry的贡献者榜单里——毕竟这就是你的最终目标了，不是吗？

BugTD commented:
你从未回答过哪怕一个关键问题

aaa-chinese commented:
> > 我相信你穿越到战锤40K世界一定会成为一名绿皮，因为你总是能发挥“我寻思”的力量而拿不出真实存在的证据
> 
> ~我相信你穿越到异世界一定会成为一名炮灰，因为你总是跟随大流而看不出主角有多强。~

well then, from this sentence alone i can feel a strong sense of self-oriented personality. i won't make more comments about it, letting others judge you is enough.

Dont-7064 commented:
> > 你瞎改很光荣，是吧
> 
> 我已经给了很多需要翻译的理由，这个 PR 即使不通过，未来也会有人根据这个会修的。

没有傻逼会跟你一样瞎改这东西

BugTD commented:
# PLEASE CLOSE THIS PR
这个issue已经几乎演变为人身攻击

Routerteaching-179 commented:
> > 你瞎改很光荣，是吧
> 
> 我已经给了很多需要翻译的理由，这个 PR 即使不通过，未来也会有人根据这个会修的。

会有人接手你这做的跟屎一样的盘子吗

AWA-SUBARU-Deisu commented:
> > > > 我相信你穿越到战锤40K世界一定会成为一名绿皮，因为你总是能发挥“我寻思”的力量而拿不出真实存在的证据
> > > 
> > > 
> > > ~我相信你穿越到异世界一定会成为一名炮灰，因为你总是跟随大流而看不出主角有多强。~
> > 
> > 
> > 我是真的想不通你为什么会把自己当成主角，你现在难道不是受人批判的魔王吗？
> 
> 还是说你把自己当成什么灵根尽废被他人唾弃的绝世天才了，你是觉得自己有系统爹吗？

又或者是说你其实真的有系统，但是有一次任务奖励是让你目中无人？

AWA-SUBARU-Deisu commented:
> ### please do not turn the discussion into personal attacks.
> ### 请不要将这个讨论上升到人身攻击。

他的所作所为不配获得平等的待遇

Routerteaching-179 commented:
> ### please do not turn the discussion into personal attacks.
> ### 请不要将这个讨论上升到人身攻击。
> i hope the discussion here should be rational at least, and not be fully driven by emotional outbursts. attacking each other does no use to this current situation. i have to admit itflake is not a cooperative person to begin with, but i really wish he could realize his faults thoroughly and sincerely, instead of getting abused by the people.

他要是听我们一点还会骂他吗

aaa-chinese commented:
> 他在这之前就已经开启过一个类似的pr，几乎所有中国区玩家都反对他做出的改动，但他仍一意孤行 He had already initiated a similar PR before this, and almost all Chinese players opposed the changes he made, but he still insisted on going his own way
yes, i am totally aware of them. his translation do have problems, and his reply is as ignorant as always.
even with that, i still don't want to just attack him with strong languages, at least this is my way of dealing this kind of things.

RA2EXE commented:
慕名而来，见证大唐盛世，丢脸丢到国外（

woodjecket commented:
ity您好，我觉得友好交流还是可以进行的，只要你先把pr撤了，这些翻译都可以谈。据我观察，你的一些翻译实际上是可取的，比如“抵御”确实比“撑过”好一些，但是现在感觉你也有一部分翻译并不可取，并不适合合并，比如“高级”改“大型”不妥，英文原文即*Advanced*而非key里的*large*，这种key和英文不符的情况在所难免，可能你也犯了望名生义的错误；而且你的行为有一点过火了，开这么多pr有点不合时宜，其他语种的翻译者都会自己成立群聊讨论然后给出统一版本，为什么我们简中就必须在pr里吵呢，如果你的pr真的被合并了，我想可能会激起大家的怒火；而且你已经陷入了一种思维的陷阱当中，感觉大家越是反对你就越证明你的对的，或者是有一种“真理在手何足惧”的感觉。

kJNCwBUq commented:
> ity您好，我觉得友好交流还是可以进行的，只要你先把pr撤了，这些翻译都可以谈。据我观察，你的一些翻译实际上是可取的，比如“抵御”确实比“撑过”好一些，但是现在感觉你也有一部分翻译并不可取，并不适合合并，比如“高级”改“大型”不妥，英文原文即_Advanced_而非key里的_large_，这种key和英文不符的情况在所难免，可能你也犯了望名生义的错误；而且你的行为有一点过火了，开这么多pr有点不合时宜，其他语种的翻译者都会自己成立群聊讨论然后给出统一版本，为什么我们简中就必须在pr里吵呢，如果你的pr真的被合并了，我想可能会激起大家的怒火；而且你已经陷入了一种思维的陷阱当中，感觉大家越是反对你就越证明你的对的，或者是有一种“真理在手何足惧”的感觉。

对此我们也感到困惑，实际上我们简中有自己的翻译讨论群。
但是此pr提交者事实上拒绝在翻译群进行交流，在discord上也是如此，所有ping他的尝试均没有回应。
